src/components/course/tests/CourseRunCard.test.jsx

```diff
@@ -17,13 +17,17 @@ import {
 import CourseRunCard from '../CourseRunCard';
 import { CourseContextProvider } from '../CourseContextProvider';
 import { UserSubsidyContext } from '../../enterprise-user-subsidy';
+import SubsidyRequestsContextProvider from '../../enterprise-subsidy-requests/SubsidyRequestsContextProvider';
 
 const COURSE_UUID = 'foo';
 const COURSE_RUN_START = moment().format();
 const COURSE_WEEKS_TO_COMPLETE = 1;
 const DATE_FORMAT = 'MMM D';
 const COURSE_ID = '123';
+
 jest.mock('../enrollment/EnrollAction', () => ({ enrollLabel }) => (<>{enrollLabel}</>));
+jest.mock('../../../config');
+// jest.mock('../../enterprise-subsidy-requests/data/hooks');
```

Comment:
nit: remove this line if it's unneeded (unless it's commented out for a reason?)